### DISCUSSION TITLE
How to kill a player without any death message?

### DISCUSSION THREAD

**Discussion by GoldenGamerLP** [Minestom/Minestom#1088] [Help] [answered]
So i wanna kill the player without any death message, but the player has in the method of killing, code that sends a death message to everyone, is there a way to bypass that? Or do i need to fork minestom?

**Comment by kiip1** [comment, marked as answer]
Use `PlayerDeathEvent.setChatMessage(null)`

(Question answer from #1072)

**Reply by ryleu** [reply]
Could you do this on the specific player’s event node?

**Reply by kiip1** [reply]
Yes, just add the listener on the player node
